### ISSUE TITLE
VS2017 seems don't solve key with - in ligatures mode

### ISSUE THREAD

**Issue by MarcoDruetta** [FortAwesome/Font-Awesome#15593]
Hi, I'using your font in Visual Studio 2017 in a WPF project. It works fine except that the composed key containing the character '-'. 
There is a solution?

![example](https://user-images.githubusercontent.com/25101248/65511334-3b8de280-ded7-11e9-91c3-66fac8890533.png)

**Comment by maki-chan** [comment]
This also happens when using the font in a WPF application. The current workaround is to use the unicode character instead.

**Comment by MarcoDruetta** [comment]
It doesn't work for me. 

I tried to replace the charactere '-' with '\u002D' directly in XAML:

`Text="chevron\u002Ddown" `

and also tried to replace it in code-behind:

```
char minus = '\u002D';
btnSortByName.Text = "chevron" + minus + "down";
```

Any other suggestion?

**Comment by maki-chan** [comment]
No, not the Unicode dash/minus character. I mean the glyph character. For example, the chevron-down icon has the unicode representation of `f078` as can be seen here: https://fontawesome.com/icons/chevron-down?style=regular

In XAML, you can for example do the following:

`<Button x:Name="btnSortByName">&#xf078;</Button>`

Or use the `\uf078` variant in code, as you tried for the minus/dash character.

**Comment by tagliala** [comment]
Sorry for the late reply, I cannot help

Leaving this open for feedback from the community

PS: does the dash work with Font Awesome 5.6.3, let's say with `cash-register`?

**Comment by MarcoDruetta** [comment]
> No, not the Unicode dash/minus character. I mean the glyph character. For example, the chevron-down icon has the unicode representation of f078 as can be seen here: https://fontawesome.com/icons/chevron-down?style=regular
> In XAML, you can for example do the following:
> <Button x:Name="btnSortByName">&#xf078;</Button>
> Or use the \uf078 variant in code, as you tried for the minus/dash character.

Good!
It works with this workaround!
Thank you
